src/app/common/components/errors/system-message/system-message.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input } from '@angular/core';
```

Comment:
It should not be placed inside `common/components/errors/system-message` but inside `common/components/system-message`. It will be used not only for errors but also for other messages (warning, info, success).